docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+# Updating the capability level of a channel
+
+*Audience: network administrators, node administrators*
+
+If you're not familiar with capabilities, check out [Capabilities](./capabilities_concept.html)
+before proceeding, paying particular attention to the fact that **node versions
+must be upgrading to the required level for all affected nodes before enabling
+capabilities**.
+
+For information about any new capability levels in the latest release of Fabric,
+check out [Upgrading your components](./upgrading_your_components.html#Capabilities).
+
+Note: when we use the term “upgrade” in Hyperledger Fabric, we’re referring to changing
+the version of a component (for example, going from one version of a binary to
+the next version). The term “update,” on the other hand, refers not to versions
+but to configuration changes, such as updating a channel configuration or a
+deployment script. As there is no data migration, technically speaking, in Fabric,
+we will not use the term "migration" or "migrate" here.
+
+## Prerequisites and considerations
+
+If you haven’t already done so, ensure you have all of the dependencies on your
+machine as described in [Prerequisites](./prereqs.html). This will ensure that
```

Comment:
There's a lot of prereqs that are not needed... in fact which specific prereq is required?

Reply:
Good question. I'm actually not sure.

Reply:
Removed the prereq section.